site/en/docs/extensions/mv3/getstarted/tut-tabs-manager/index.md

```diff
@@ -0,0 +1,382 @@
+move them into the current window.
+
+{% Label %}popup.js:{% endLabel %}
+
+```js
+...
+const button = document.querySelector("button");
+button.addEventListener("click", async () => {
+  const group = await chrome.tabs.group({ tabIds: tabs.map(({ id }) => id) });
+  await chrome.tabGroups.update(group, { title: "DOCS" });
+});
+```
+
+The [TabGroups][api-tabgroups] API allows the extension to name the group and choose a background
+color. Add the `“tabGroups”` permission to the manifest by adding the following code:
+
+{% Label %}manifest.json:{% endLabel %}
+
+```json
+{
+  ...
+  "permissions": [
+    "tabGroups"
+  ]
+}
+```
+
+## Test that it works {: #try-out }
+
+Verify that the file structure of your project looks like the following: 
+
+```text
+└── Tabs-manager/
+    ├── manifest.json
+    ├── popup.js
+    ├── popup.css
+    ├── popup.html
+    └── images/
+        ├── icon-16.png
+        ├── icon-32.png
+        ├── icon-48.png
+        └── icon-128.png
+```
+
+### Load your extension locally {: #locally }
+
+To load an unpacked extension in developer mode, follow the steps in [Development
+Basics][doc-dev-basics-unpacked].
+
+### Open a few documentation pages {: #open-sites}
+
+Open the following docs in different windows:
+
+- [Design the user interface][doc-ui]
+- [Discovery on the Chrome Web Store][cws-discovery]
+- [Extension development overview][doc-devguide]
+- [Manifest file format][doc-manifest]
+- [Publish in the Chrome Web Store][cws-publish]
+- [Welcome to Chrome Extension Development][doc-welcome]
+
+
+Click on the popup. It should look like this:
+
+<figure>
+{% Img src="image/BhuKGJaIeLNPW9ehns59NfwqKxF2/8q5ps3dqw4p2BOZRpIJT.png", 
+alt="Tabs Manager extension popup", width="600", height="230", class="screenshot" %}
+  <figcaption>
+  Tabs Manager extension popup
+  </figcaption>
+</figure>
+
+## Potential enhancements {: #challenge }
+
+Based on what you’ve learned today, try to implement any of the following:
+
+- Improve the popup stylesheet.
+- Change the color and title of the tab group.
+- Manage the tabs of another documentation site.
+
+## Keep building! {: #continue }
+
+Congratulations on finishing this tutorial 🎉. Continue developing your skills by completing other
+tutorials on this series:
+
+| Extension                        | What you will learn                                                    |
+|----------------------------------|------------------------------------------------------------------------|
+| [Reading time][tut-reading-time] | To insert an element on every page automatically.                      |
+| [Focus Mode][tut-focus-mode]     | To run code on the current page after clicking on the extension action. |
+
+## Continue exploring
+
+We hope you enjoyed building these Chrome extensions and are excited to continue your Chrome
+development learning journey. We recommend the following learning path:
```

Comment:
This paragraph sounds like it's signing off for the entire series of tutorials. Is that the intent?

Given that the samples can be completed in any order, I think it may be better to include this section in all of the tutorials.

Reply:
Done! 👍